admin_guide/managing_networking.adoc

```diff
@@ -545,8 +545,12 @@ endif::openshift-origin[]
 moves it into the pod's network project before starting the *egress-router*
 container. Preserve the quotation marks around `"true"`. Omitting them results
 in errors. To create the Macvlan interface on a network interface other than the primary one, set the annotation value to the name of that interface. For example, `eth1`.
-<2> IP address from the physical network that the node is on and is
-reserved by the cluster administrator for use by this pod.
+<2> IP address from the physical network that the node is on and is reserved by the
+cluster administrator for use by this pod. Optionally, you can include the
+subnet length (the `/24`) so that a proper route to the local subnet can be set
```

Comment:
To avoid parentheses, s/length (the `/24`)/length, the `/24` suffix, 
s/set up/set